circus-train-core/src/main/java/com/hotels/bdp/circustrain/core/replica/hive/RenameTableOperation.java

```diff
@@ -40,20 +40,27 @@ public RenameTableOperation(DropTableService dropTableService) {
    * renamed.
    */
   public void execute(CloseableMetaStoreClient client, Table from, Table to) throws Exception {
-    LOG
-        .info("Renaming table {}.{} to {}.{}", from.getDbName(), from.getTableName(), to.getDbName(),
-            to.getTableName());
+    LOG.info("Renaming table {}.{} to {}.{}", from.getDbName(), from.getTableName(), to.getDbName(), to.getTableName());
     Table fromTable = client.getTable(from.getDbName(), from.getTableName());
     Table toTable = client.getTable(to.getDbName(), to.getTableName());
+    String fromDatabaseName = fromTable.getDbName();
     String fromTableName = fromTable.getTableName();
     String toTableName = toTable.getTableName();
```

Comment:
Worth also having `String toDatabaseName = toTable.getDbName()` ?  
toTable.getDbName() is called quite a few times

Reply:
sure